### DISCUSSION TITLE
Create a Next.js App

### DISCUSSION THREAD

**Discussion by asd123123123123123123** [vercel/next.js#36846] [Help]
hello im a student and i'm practicing Create a Next.js App tutorial. I used git bash and follow the command. after i make the Welcome to [Next.js!](https://nextjs.org/) page i can't edit the page. i don't know the meaning of 'Open pages/index.js with your text editor.' please help me. i'm a beginner.  
![image](https://user-images.githubusercontent.com/105400138/167970607-4ad61837-472d-426f-829d-3dbed639be58.png)

**Comment by kahfiii** [comment]
maybe the meaning is you can open file "index.js" in your text editor/code editor like sublime text ,visual studio code or atom

**Reply by icyJoseph** [reply]
Exactly this.

You need to open the `nextjs-blog` folder, with an IDE (Integrated Development Environment) or a simple Text Editor. I'd recommend VSCode as having one of the lowest effort entry points. Once you make changes there, they'll hot reload and update the page.
